CHANGELOG.md

```diff
@@ -1,247 +1,515 @@
+  `StatefulSetAutoDeletePVC`,
+  `PodProbeMarkerGate` ([#1919](https://github.com/openkruise/kruise/pull/1919), [@ABNER-1](https://github.com/ABNER-1))
+- Update Kubernetes dependency to v1.30.9 and Golang to
+  v1.22 ([#1896](https://github.com/openkruise/kruise/pull/1896), [@ABNER-1](https://github.com/ABNER-1))
+
+### Key Features
+
+- Support StatefulSet volume
+  expansion ([#1674](https://github.com/openkruise/kruise/pull/1674), [#1714](https://github.com/openkruise/kruise/pull/1714), [@ABNER-1](https://github.com/ABNER-1))
+- Support in-place resource
+  resizing ([#1353](https://github.com/openkruise/kruise/pull/1353), [#1866](https://github.com/openkruise/kruise/pull/1866), [@LavenderQAQ](https://github.com/LavenderQAQ), [@ABNER-1](https://github.com/ABNER-1))
+
+- Add Helm pre-delete hook to preserve Kruise CRs during
+  uninstallation ([#1843](https://github.com/openkruise/kruise/pull/1843), [@AiRanthem](https://github.com/AiRanthem))
+
+### Performance Improvements
+
+- Optimize CA bundle updates to reduce unnecessary
+  changes ([#1717](https://github.com/openkruise/kruise/pull/1717), [@zmberg](https://github.com/zmberg))
+- Add disableDeepCopy for
+  BroadcastJob ([#1696](https://github.com/openkruise/kruise/pull/1696), [@Prepmachine4](https://github.com/Prepmachine4))
+
+### Resilience Enhancement
+
+- Extend PodProbeMarker to serverless
+  pods ([#1875](https://github.com/openkruise/kruise/pull/1875), [@zmberg](https://github.com/zmberg))
+- Support adaptive scheduling strategy for
+  UnitedDeployment ([#1720](https://github.com/openkruise/kruise/pull/1720),  [@AiRanthem](https://github.com/AiRanthem))
+
+### Other Notable Changes
+
+#### Advanced Workload
+
+- Add lifecycle hooks and tests for Advanced
+  StatefulSet ([#1858](https://github.com/openkruise/kruise/pull/1858), [@mingzhou.swx](https://github.com/mingzhou.swx), [@ABNER-1](https://github.com/ABNER-1))
+- Add range-based reserveOrdinals support for Advanced
+  StatefulSet ([#1873](https://github.com/openkruise/kruise/pull/1873), [@AiRanthem](https://github.com/AiRanthem))
+- Redefined partition semantics to represent non-updated pod
+  count ([#1819](https://github.com/openkruise/kruise/pull/1819), [@ABNER-1](https://github.com/ABNER-1); [#1751](https://github.com/openkruise/kruise/pull/1751), [@zybtakeit](https://github.com/zybtakeit), [@ABNER-1](https://github.com/ABNER-1))
+
+#### Sidecar Management
+
+- Enable version-aware injection for
+  SidecarSet ([#1689](https://github.com/openkruise/kruise/pull/1689), [@AiRanthem](https://github.com/AiRanthem)); and
+  introduce Partial strategy for SidecarSet
+  injection ([#1856](https://github.com/openkruise/kruise/pull/1856), [@AiRanthem](https://github.com/AiRanthem))
+- Refine SidecarSet initContainer
+  handling ([#1719](https://github.com/openkruise/kruise/pull/1719), [@zmberg](https://github.com/zmberg))
+
+#### Multi-domain management
+
+- Add WorkloadSpread support for job-like
```

Comment:
"Add WorkloadSpread support for AI workload like TFJob" 

and i think it is also a key feature